src/fixml/modules/workflow/parse.py

```diff
@@ -91,9 +90,14 @@ def get_completeness_score(self, score_format: str = 'fraction', verbose: bool =
                 print()
                 print(f"WARNING: `n_files_tested` is not unique!")
                 print()
-            print(f'Score: {score}')
+            print(f'Result: {num_items_satisfied} items are satisfied out of {num_items}')
+            print(f'Score: {fraction_satisfied*100:.1f}%')
             print()
-        return score
```

Comment:
This return value is needed when exporting the reoprt, see line 131 (or line 127 before the change)

Reply:
https://github.com/UBC-MDS/fixml/blob/e9fa5babbd20ab219b0d64067d0a44bfc6877a06/src/fixml/modules/workflow/parse.py#L131

Reply:
@SoloSynth1 yes, i think it returns the same thing. I just changed the variable name.

Reply:
Oh ok my bad! Resolving this.